### ISSUE TITLE
[Feature]: Add Twitter/X link to home page

### ISSUE THREAD

**Issue by gogwilt** [Ironclad/rivet#297]
### Feature Request

The rivet_ts Twitter/X handle is hard to find, so it would be helpful to add it to the homepage.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by codemile** [comment]
Agreed, we should add the X link now, but we'll probably have to remove it in 12 months. Hahaha....
